sql/ddl.sql

```diff
@@ -1666,3 +1666,110 @@ VALUES ('Bruges når et punkt og tilhørende geometri nedlægges.', 'punkt_nedla
 
 INSERT INTO eventtype (beskrivelse, event, eventtypeid)
 VALUES ('Bruges til at tilføje fritekst-kommentarer til sagen i tilfælde af at der er behov for at påhæfte sagen yderligere information, som ikke passer i andre hændelser. Bruges fx også til påhæftning af materiale på sagen.', 'kommentar', 9);
+
+-- udvidelse - flyttes "på plads" senere
+-------------------------------------------------------
+
+-- PUNKTGRUPPE
+-------------------------------------------------------
+-- Usecases:
+-- * Opret punktgruppe ud fra n punktid'er
+-- * Tilføj punkter til punktgruppe, via punktgruppe_punkt
+-- * Punkter fjernes IKKE fra punktgruppe
+-- * Ingen versionering - punktgruppe kan ift reg.fra/til og sagsevents
+--   oprettes og lukkes, ikke opdaterres
+CREATE TABLE punktgruppe (
+  objektid INTEGER GENERATED ALWAYS AS IDENTITY (
+    START WITH
+    1 INCREMENT BY 1 ORDER NOCACHE
+  ) PRIMARY KEY,
+
+  registreringfra TIMESTAMP WITH TIME ZONE NOT NULL,
+  registreringtil TIMESTAMP WITH TIME ZONE,
+  sagseventfraid VARCHAR2(36) NOT NULL,
+  sagseventtilid VARCHAR2(36),
+
+  jessenpunktid VARCHAR2(36) NOT NULL, -- Det fastholdte punkt i gruppen
+  -- har vi også brug for den specifikke kote for jessenpunktet? Fx
+  -- jessenkoordinatid INTEGER, med foreign key på koordinat.objektid
```

Comment:
Jeg har tilføjet den under `tidsserie`, da principielt ikke er noget i vejen for at bruge den samme punktgruppe til flere formål, fx til tidsserier i DVR90 samt et lokalt referenceniveau. Her vil det være nyttigt let at kunne finde den kote de andre kote er relative til.

Reply:
Er der behov for `jessenpunktid` i de andre 'punktgruppe'-scenarier?

Reply:
Det kan jeg godt forestille mig, ja. Umiddelbart vil jeg godt mene der kan være fidus i at have både en DVR90- og lokalreference-tidsserie på en vandstandsmåler men det er mest gætværk.

Reply:
Hvis vi kan kvalificere et gæt, kan det bruges som udgangspunkt fremover;) Men OK. Jeg lukker denne her:)